iothub/device/src/IotHubBaseClient.cs

```diff
@@ -48,6 +48,8 @@ public abstract class IotHubBaseClient : IDisposable
                 ? iotHubClientOptions.Clone()
                 : new();
 
+            Console.WriteLine(_clientOptions);
```

Comment:
You can't just write out an object to console. You won't get what you'd want.

You need to either print out specific properties (which themselves are primitive types) or override the `ToString()` method on that object to print out what you want.

So, what are you hoping to do with this call?

Reply:
Forgot to delete it, will delete